doc/man3/DEFINE_STACK_OF.pod

```diff
@@ -31,7 +32,7 @@ sk_TYPE_dup, sk_TYPE_deep_copy, sk_TYPE_set_cmp_func - stack container
  TYPE *sk_TYPE_value(const STACK_OF(TYPE) *sk, int idx);
  STACK_OF(TYPE) *sk_TYPE_new(sk_TYPE_compfunc compare);
  STACK_OF(TYPE) *sk_TYPE_new_null(void);
- int sk_TYPE_reserve(STACK_OF(TYPE) *sk, size_t n);
+ int sk_TYPE_reserve(STACK_OF(TYPE) *sk, int n);
```

Comment:
Why?

Reply:
Because that's what OPENSSL_sk_reserve accepts. It's harmonization with code.

Reply:
Ah!  Thanks for the reminder

Reply:
Internally the stack structure uses an `int` for the number of elements present.  This is exposed by the call `sk_TYPE_num` which returns an `int`.  Changing to a `size_t` makes sense to me but it is an API change.

Reply:
In fact, next time when API could be changed, IMHO the `new` APIs of stack could be squashed into one, as something like `sk_TYPE_new(int n, sk_TYPE_compfunc compare)`. If `n == 0`, then it equals to `sk_TYPW_new_null`...

Reply:
Just to clarify, this is wrong in the current documentation, and the documentation is fixed. Maybe you can move that fix to a different pull request so we can merge that to other branches?

Reply:
Or a separated commit inside this PR? and thus you can cherry-pick that commit...

Reply:
That works too of course